src/app/content/highlights/hooks/index.ts

```diff
@@ -23,6 +26,8 @@ export default [
   openMyHighlightsHook,
   printHighlightsHook,
   loadMoreHook,
+  actionHook(closeMyHighlights, closeModal),
```

Comment:
if the url were to be replaced with a different modal name, you'd still want to close it, so i think this needs to work more similarly to the open one

Reply:
Since opening a new modal would dispatch `locationChange`, I think closing the current modal would be handled by the reducer:
```ts
    case getType(locationChange): {
      const summaryShouldBeOpen =
        getParamFromQuery(modalQueryParameterName, action.payload.location.search) === modalUrlName
        && action.payload.action === 'PUSH';

      return {...state, summary: {...state.summary, open: summaryShouldBeOpen}};
    }
```

Reply:
However current implementation calls `history.goBack` when `closeModal` is dispatched and I think this will not work when we will implement adding a filters to the url - unless we will update the url with `replace` action

Reply:
i'm not sure what i was talking about here, i think i wanted to wrap the goBack in a check to see if the correct modal was currently in the url, but i'm not sure when it would matter

Reply:
I'm not sure if we should use `goBack` here at all. Consider the case like changing the url when navigating through the questions:
1. Open PQ / modal=PQ
2. Go to first question / modal=PQ&question=1
3. Got next question / modal=PQ&question=2
4. Close the modal / .goBack will go to the question=1

Reply:
Currently, we don't support it. I guess we can go back to this issue when we will implement this